app/models/user.rb

```diff
@@ -249,8 +250,9 @@ def set_remember_fields
   end
 
   def calculate_score
```

Comment:
We are currently not using this at all, despite it being present. This allows us to start caring about user score for spam prevention or other quality control.